xtask/src/commands/bump_version.rs

```diff
@@ -0,0 +1,184 @@
+use std::{fs, path::Path};
+
+use anyhow::{Context, Result, bail};
+use clap::Args;
+use semver::Prerelease;
+use strum::IntoEnumIterator;
+use toml_edit::{Item, Value};
+
+use crate::{Package, Version};
+
+#[derive(Debug, Args)]
+pub struct BumpVersionArgs {
+    /// How much to bump the version by.
+    ///
+    /// If the version is a pre-release, this will just remove the pre-release
+    /// version tag.
+    #[arg(value_enum)]
+    amount: Version,
+    /// Pre-release version to append to the version.
+    ///
+    /// If the package is already a pre-release, this will ignore `amount`. If
+    /// the package is not a pre-release, this will bump the version by
+    /// `amount` and append the pre-release version as `<pre>.0`.
+    #[arg(long)]
+    pre: Option<String>,
+    /// Package(s) to target.
+    #[arg(value_enum, default_values_t = Package::iter())]
+    packages: Vec<Package>,
+}
+
+pub fn bump_version(workspace: &Path, args: BumpVersionArgs) -> Result<()> {
+    // Bump the version by the specified amount for each given package:
+    for package in args.packages {
+        bump_crate_version(workspace, package, args.amount, args.pre.as_deref())?;
```

Comment:
The only code change in this PR is this one rename, to avoid the collision. The implementation has been renamed from `bump_version` to `bump_crate_version`.